internal/cmd/common/common.go

```diff
@@ -77,12 +80,7 @@ func LoadPublicKey(key string) (*tuf.Key, error) {
 			},
 		}
 	default:
-		kb, err := os.ReadFile(key)
-		if err != nil {
-			return nil, err
-		}
-
-		keyObj, err = tuf.LoadKeyFromBytes(kb)
+		keyObj, err = ssh.NewKeyFromFile(key)
```

Comment:
BREAKING CHANGE: we also end up with new key IDs for the same keys as before, I think.